tests/foreman/cli/test_host_collection.py

```diff
@@ -119,7 +119,7 @@ def test_positive_create_3(self):
 
     @skip_if_bug_open('bugzilla', 1214675)
     @tier1
-    def test_create_hc_with_unlimited_content_hosts(self):
+    def test_positive_create_with_unlimited_chosts(self):
```

Comment:
`unlimited-content-hosts` is parameter name, imo parts of parameter name shouldn't be shortened